qa/tasks/cephfs/test_snap_schedules.py

```diff
@@ -588,8 +588,8 @@ def remove_snapshots(self, dir_path, sdn):
             self.mount_a.run_shell(['rmdir', snapshot_path])
 
     def get_snap_dir_name(self):
-        from tasks.cephfs.fuse_mount import FuseMount
-        from tasks.cephfs.kernel_mount import KernelMount
+        from .fuse_mount import FuseMount
```

Comment:
How come this fails only intermittently then?

Reply:
this, I have no idea about
I humbly request Python experts to lend a hand here

Reply:
Its likely that something is messing up PYTHONPATH causing this to error out.